lib/systems/include/Controllers/CASEControllerWrapper.h

```diff
@@ -9,10 +9,14 @@ class TorqueControllerCASEWrapper : public virtual Controller
 {
 public:
     TorqueControllerCASEWrapper() = delete;
-    
+    /// @brief makes CASE system apart of Controller hierarchy for use in TC Mux
+    /// @param case_instance 
```

Comment:
Is there a reason you don't include a type for your @param tag here, but you do everywhere else? Typically, types are unnecessary for @param tags. If the type is relevant to that particular argument, then you may include it.